src/SemCor_DataDeps.jl

```diff
@@ -5,6 +5,7 @@ for (ver, checksum) in [("1.6", "16814254fe194d55a2fcc24858aa76d71de3c49e495bd98
             ("1.7", "0495577ac3a87c2a64fe6189798ea046de0f44943dfb7b60fe38cf648d34c421"),
             ("1.7.1", "70b9eb7ca0dc9d67655f9d671d40be10aeff490f0bea4f10cb1946127b74c102"),
             ("2.0", "93fbae725f0125dedb7369403fda1dace85b2dcd8a523ed80af23e863b18ef2c"),
+	    ("2.1", "0714f07dbcb84a215d668f3ee85892fa8fa4a8154439662eb7529413367b8f56"),
```

Comment:
The indentation seems a bit off. Perhaps because of the usage of both spaces and tabs.

Otherwise, LGTM.

Reply:
Resolved in 04bb500